static/js/stat_var_hierarchy/stat_var_hierarchy.tsx

```diff
@@ -250,85 +253,85 @@ export class StatVarHierarchy extends React.Component<
     );
   }
 
-  private fetchData(): void {
+  private async fetchData(): Promise<void> {
     loadSpinner(SV_HIERARCHY_SECTION_ID);
     const entityList = this.props.entities.map((entity) => entity.dcid);
-    const allPromises: Promise<
-      string[] | StatVarGroupInfo[] | StatVarGroupNodeType
-    >[] = [];
-    allPromises.push(
-      axios
-        .post("/api/variable-group/info", {
-          dcid: ROOT_SVG,
-          entities: entityList,
-          numEntitiesExistence: this.props.numEntitiesExistence,
-        })
-        .then((resp) => {
-          return resp.data["childStatVarGroups"];
-        })
-    );
-    if (globalThis.dataSourceDcid) {
-      allPromises.push(
+    const variableGroupInfoPromises: Promise<StatVarGroupNodeType>[] =
+      STAT_VAR_HIERARCHY_CONFIG.nodes.map((statVarHierarchyConfigNode) =>
         axios
           .post("/api/variable-group/info", {
-            dcid: ROOT_SVG,
-            entities: [globalThis.dataSourceDcid],
+            dcid: statVarHierarchyConfigNode.dcid,
+            entities: [
```

Comment:
maybe define `entities` with conditional statement to make it more explicit

Reply:
Updated